packages/nextjs/components/scaffold-eth/Contract/SkeletonContractUI.tsx

```diff
@@ -0,0 +1,104 @@
+const SkeletonP = ({ style }: { style?: string }) => {
+  return <p className={`my-0 mt-2 w-24 h-3 bg-gray-500 animate-pulse rounded-md ${style}`}></p>;
+};
+
+const SkeletonDiv = ({ style }: { style?: string }) => {
+  return <div className={`h-6 w-6 bg-gray-400 animate-pulse rounded-md ${style}`}></div>;
+};
+
+const SkeletonInputButton = () => {
+  return (
+    <div className="flex flex-col">
+      <SkeletonDiv style="!w-full !rounded-2xl !h-7 mb-2" />
```

Comment:
hmm, we probably need props for width/height/rounded. (See https://github.com/scaffold-eth/se-2/pull/178#issuecomment-1435691908 first)

Reply:
Yeah wanted to find a better way to handle this, let's get confirmation from @carletex if we keeping this or not 🙌